prow/config.yaml

```diff
@@ -146,7 +244,6 @@ presubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
-
   - name: pull-kubernetes-bazel-test
```

Comment:
eh,  I think `bazel-test` chains after `bazel-build` and parallel to kubeadm

Reply:
`bazel-build` and `bazel-test` should be in parallel